tools/bowtie2/bowtie2_wrapper.xml

```diff
@@ -147,8 +149,12 @@
         #end if    
         
         ## view/sort and output BAM file
-        | samtools view -Su - | samtools sort -o - - > $output
-        
+        #if ( str( $analysis_type.analysis_type_selector ) != "full" or str( $analysis_type.sam_opt ) != "true" ):
+          | samtools view -Su - | samtools sort -o - - > $output_1
+        #else
+          > $output_2
+        #end if
+
```

Comment:
Shouldn't these greater than signs for redirection be HTML escaped (or using CDATA)?